test/e2e/utils/test_context.go

```diff
@@ -243,10 +243,18 @@ func (t *TestContext) Destroy() {
 	//nolint:gosec
 	// if image name is not present or not provided skip execution of docker command
 	if t.ImageName != "" {
-		cmd := exec.Command("docker", "rmi", "-f", t.ImageName)
-		if _, err := t.Run(cmd); err != nil {
-			warnError(err)
+		// Remove white space from image name
+		trimmedImageName := strings.Trim(t.ImageName, "")
+		// Check trimmedimage name is empty
+		if len(trimmedImageName) < 0 {
+			fmt.Println("Image not set, skip cleaning up of docker image")
```

Comment:
Also, see the above lint error 
Run make lint to test your code locally and see that you need to fix the lint issues as well prior we get merged 
Some issues will be fixed by running make lint-fix .. if not, they you can search by the error and see what you need to do to fix them.

Feel free to ping if you need to.